src/state.py

```diff
@@ -68,3 +68,5 @@
 
 thisapp = None
 """Singleton instance"""
+
+isKivyworking = False
```

Comment:
rename

Reply:
now i am renaming it with kivyAppWorking